### PR TITLE
add 2018 champions post

### PR DESCRIPTION
**Pull request by andrea-p**

### CODE REVIEW

File: posts/2018-04-10-announcing-new-2018-serverless-champions.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+[Read the full interview](https://serverless.com/community/champions/alex-casalboni/), and find Alex on:
+
+[Twitter](https://twitter.com/alex_casalboni) | [GitHub](https://github.com/alexcasalboni) | [site](https://blog.alexcasalboni.com/)
+
+### Frank Schmid
+
+<img align="left" src="https://s3-us-west-2.amazonaws.com/assets.site.serverless.com/images/champions/2018/frank-schmid.jpg" width="210px">
+
+Frank is a core maintainer of the Serverless Framework, and is incredibly proactive about stepping in on GitHub issues, and joining discussions on the forums and in Serverless Slack channels. He uses the Serverless Framework daily for his job at Stashimi.
```
Comment (HyperBrain) [inline posts/2018-04-10-announcing-new-2018-serverless-champions.md]: You could also mention, that I'm author/maintainer of `serverless-webpack` 😄

Comment (rupakg) [reply posts/2018-04-10-announcing-new-2018-serverless-champions.md]: done.

**Comment by rupakg** [comment]
schedule(04/10/2018 6:00 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **04/10/2018 at 6:00am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/560#issuecomment-379975733)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**